gcn/notices/neutrino_telescope/NeutrinoAlert.schema.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/main/gcn/notices/neutrino_telescope/NeutrinoAlert.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "title": "NeutrinoAlert",
+  "description": "Neutrino alert information common to all neutrino telescope alerts",
+  "allOf": [
+    {
+      "$ref": "../core/Event.schema.json",
+      "description": "Unique per-alert identification information from source telescope"
+    },
+    {
+      "$ref": "../core/Alert.schema.json",
+      "description": "Alert information (alert_datetime and alert_type)"
+    },
+    {
+      "$ref": "../core/Reporter.schema.json",
+      "description": "Reporting telescope ID information"
+    },
+    {
+      "$ref": "../core/Statistics.schema.json",
+      "description": "Alert signficance measures (FAR, p_astro)"
+    },
+    {
+      "$ref": "../core/Localization.schema.json",
+      "description": "Alert sky localization information (RA/DEC, error)"
+    },
+    { "$ref": "../core/DateTime.schema.json" }
+  ],
+  "properties": {
+    "number_of_events": {
+      "description": "Number of detected events that contributed to alert",
+      "type": "number"
+    },
+    "analysis_pipeline": {
+      "description": "Name of analysis pipeline used to generate alert",
+      "type": "string"
+    },
+    "alert_topology": {
+      "description": "Topology of the alert (e.g. track, cascade, multiplet...)",
```

Comment:
```suggestion
      "description": "Topology of the alert",
```

Reply:
It's an enum, so doesn't need to be listed also in the description.

Reply:
removed